src/components/InputCard.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react'
+import styled from 'styled-components'
+import Textarea from 'react-textarea-autosize'
+import Card from './Card'
+import DatePicker from 'react-datepicker'
+import 'react-datepicker/dist/react-datepicker.css'
+
+export const StyledTextarea = styled(Textarea)`
+  background: none;
+  border: 0;
+  color: white;
+  font-family: 'Niramit', sans-serif;
+  font-size: 20px;
+  margin-bottom: 14px;
+  margin-top: 14px;
+  outline: none;
+  width: 100%;
+
+  ::-webkit-input-placeholder {
+    /* Chrome/Opera/Safari */
+    color: #ffffffb0;
+  }
+  ::-moz-placeholder {
+    /* Firefox 19+ */
+    color: #ffffffb0;
+  }
+  :-ms-input-placeholder {
+    /* IE 10+ */
+    color: #ffffffb0;
+  }
+  :-moz-placeholder {
+    /* Firefox 18- */
+    color: #ffffffb0;
+  }
+`
+
+export const StyledInput = styled(DatePicker)`
+  border: 0;
+  font-family: 'Niramit', sans-serif;
+  font-size: 20px;
+  margin-top: 1px;
+  outline: none;
+  width: 100%;
+  padding-left: 10px;
+`
+
+export const Wrapper = styled.div`
+  display: flex;
+  padding: 22px;
+`
+export const StyledHeadline3 = styled.h3`
+  font-family: 'Niramit', sans-serif;
```

Comment:
Ist es vielleicht sinnvoll die Schriftart als Variable zu übergeben? Nur weil du sie an so vielen Stellen verwendest.

Reply:
Das wäre eine Überlegung wert. Vielleicht stell ich Sie auch direkt in den Global styles ein :) Danke für den Hinweis!